old-fashioned-image-build

```diff
@@ -199,11 +204,20 @@ export MIRROR=$(egrep "(archive|ports)" /etc/apt/sources.list|head -1 | \
 export MIRROR_PATH=$(egrep "(archive|ports)" /etc/apt/sources.list|head -1 | \
                cut -d' ' -f2 | cut -d'/' -f4)
 
-/usr/share/launchpad-buildd/bin/in-target override-sources-list \
-  --backend=lxd --series=$SERIES --arch=${ARCH} $LIVEFS_NAME \
-    "deb http://$MIRROR/${MIRROR_PATH} $SERIES main restricted universe multiverse" \
-    "deb http://$MIRROR/${MIRROR_PATH} $SERIES-security main restricted universe multiverse" \
-    "deb http://$MIRROR/${MIRROR_PATH} $SERIES-updates main restricted universe multiverse"
+if [ -n "$PROPOSED" ]; then
+  /usr/share/launchpad-buildd/bin/in-target override-sources-list \
+    --backend=lxd --series=$SERIES --arch=${ARCH} $LIVEFS_NAME \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES main restricted universe multiverse" \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES-proposed main restricted universe multiverse" \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES-security main restricted universe multiverse" \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES-updates main restricted universe multiverse"
+else
+  /usr/share/launchpad-buildd/bin/in-target override-sources-list \
+    --backend=lxd --series=$SERIES --arch=${ARCH} $LIVEFS_NAME \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES main restricted universe multiverse" \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES-security main restricted universe multiverse" \
+      "deb http://$MIRROR/${MIRROR_PATH} $SERIES-updates main restricted universe multiverse"
+fi
```

Comment:
To avoid the duplicated lines, can you:

1) set a variable to incude the three common lines found in both sources.list
2) if proposed is enabled, add that line to the above variable
3) have one call to in-target